docs/api/authorization.rst

```diff
@@ -0,0 +1,90 @@
+Authorization
+=============
+
+API requests which only read public data do not require authorization.
+
+API requests made as a particular user or which manage user accounts or groups
+require authorization.
+
+.. toctree::
+   :maxdepth: 1
+
+   using-oauth
+
+Access tokens
+-------------
+
+API requests which read or write data as a specific user need to be authorized
+with an access token. Access tokens can be obtained in two ways:
+
+1. By generating a personal API token on the `Hypothesis developer
+   page <https://hypothes.is/account/developer>`_ (you must be logged in to
+   Hypothesis to get to this page). This is the simplest method, however
+   these tokens are only suitable for enabling your application to make
+   requests as a single specific user.
+
+2. By :ref:`registering <registering-an-oauth-client>` an "OAuth client" and
+   :ref:`implementing <implementing-oauth-flow>` the OAuth authentication flow
+   in your application. This method allows any user to authorize your
+   application to read and write data via the API as that user.  The Hypothesis
+   client is an example of an application that uses OAuth.
+
+   See :doc:`using-oauth` for details of how to implement this method.
```

Comment:
I put the details of using OAuth in a separate doc because I think it likely that a first time reader will decide that the API token is a simpler place to get started for testing purposes.